CONVENTIONS.md

```diff
@@ -84,3 +84,13 @@ the variable.
 [enum]: https://doc.rust-lang.org/reference.html#enumerations
 [libc]: https://crates.io/crates/libc/
 [std_MaybeUninit]: https://doc.rust-lang.org/stable/std/mem/union.MaybeUninit.html
+
+## Pointer type casting
+
+We prefer [`cast()`], [`cast_mut()`] and [`cast_const()`] to cast pointer types
+over the `as` keyword because it is way more easier to change both type and 
```

Comment:
Shouldn't this be "more difficult" rather than "more easier"?  Also, for written docs prefer "much" over "way".

Reply:
Ah, right

Reply:
done